packages/cli/src/Queue.ts

```diff
@@ -19,10 +20,13 @@ export type JobQueue = Bull.Queue<JobData>;
 export interface JobData {
 	executionId: string;
 	loadStaticData: boolean;
+	returnResults?: boolean;
 }
 
 export interface JobResponse {
 	success: boolean;
+	executionResponse?: IExecutionResponse;
+	error?: ExecutionError;
```

Comment:
These changes are not necessary at this point, right? Should we consider removing?

Reply:
depends, do we plan on using it soon, or not (in which case i will remove it)

Reply:
thinking about this after our discussion, i'd like to keep the error as part of the response, because if as part of the optimization we do not pull the data from the db, it would still be useful to have the error available on main